src/commands/Configuration/configure/automod/add.ts

```diff
@@ -173,7 +177,26 @@ async function add(interaction: MessageComponentInteraction, client: BulbBotClie
 							} else appendContent = users;
 						}
 
-						if (!interaction.guild?.id) return;
+						// handle the avatar bans
+						else if (selectedCategory === "avatarHashes") {
+							const user = await client.bulbfetch.getUser(appendContent.replace(NonDigits, ""));
+							if (!user) {
+								await interaction.followUp({
+									content: await client.bulbutils.translate("config_automod_add_remove_add_fail", interaction.guild?.id, {
+										item: appendContent,
+									}),
+									ephemeral: true,
+								});
+
+								return add(i, client, selectedCategory);
```

Comment:
Why do we still call add in the fail case?

Reply:
eh good question, we can just break